metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -1554,6 +1554,7 @@ def _snowflake_clear_ocsp_cache(self):
 
     def close(self) -> None:
         super().close()
+        super(StatefulIngestionSourceBase, self).close()
```

Comment:
I think this is needed because `SnowflakeConnectionMixin.close` does not call `super().close()`. It seems slightly better to me to propagate the close call rather than directly call it here